spec/preservation_ingest/complete_ingest_spec.rb

```diff
@@ -42,7 +15,6 @@
     end
 
     it 'removes the deposit bag and updates the accessionWF when no errors are raised' do
```

Comment:
we should strike the "removes the deposit bag" part from the title now

Reply:
Struck.